website/src/css/custom.css

```diff
@@ -134,4 +161,43 @@ div[class^='announcementBar_'] {
   .footer-logo {
     margin: 2rem;
   }
-}
\ No newline at end of file
+}
+
+.button.button--accent {
+  background-color: var(--glasskube-pink);
+  color: var(--glasskube-main-light);
+}
+
+.button.button--outline {
+  position: relative;
+  color: var(--ifm-navbar-link-color);
+}
+
+.button.button--outline::before {
+  content: "";
+  position: absolute;
+  inset: 0;
+  border-radius: 8px;
+  border: 3px solid transparent;
+  background: linear-gradient(to right, var(--glasskube-main-3), var(--glasskube-pink)) border-box;
+  -webkit-mask: linear-gradient(#fff 0 0) padding-box, linear-gradient(#fff 0 0);
+  -webkit-mask-composite: xor;
+  mask-composite: exclude;
+}
+
+h1 {
+  background: linear-gradient(60deg, var(--glasskube-main-3), var(--glasskube-pink));
+  -webkit-background-clip: text;
```

Comment:
should use non-prefixed variants:
https://developer.mozilla.org/en-US/docs/Web/CSS/mask
https://developer.mozilla.org/en-US/docs/Web/CSS/mask-composite
https://developer.mozilla.org/en-US/docs/Web/CSS/background-clip